scratch/copy-mk.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+################################################################################
+# Copyright (c) 2024, National Research Foundation (SARAO)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+"""Copy a running MK correlator.
+
+This script makes use of sim_correlator.py in order to start a correlator, but
+instead of a simulated one, it gets the parameters from a live MK correlator.
+"""
+
+import argparse
+import asyncio
+import subprocess
+import sys
+from collections.abc import Callable, Sequence
+from dataclasses import dataclass
+
+import aiokatcp
+
+
+@dataclass
+class Subordinate:
+    name: str
+    control_port: int
+    monitor_port: int
+    digitiser_address: str
+    n_antennas: int
+
+
+def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("cmc_host", help="Hostname / IP address of the CMC.")
+    parser.add_argument("--cmc_port", type=int, default=7147, help="KATCP port of the CMC. [%(default)s]")
+    parser.add_argument("--target", type=str, required=True, help="Which subordinate to copy.")
+    parser.add_argument(
+        "--develop",
+        nargs="?",
+        const=True,
+        help="Pass development options in the config. Use comma separation, or omit the arg to enable all.",
+    )
+    parser.add_argument("--image-override", action="append", metavar="NAME:IMAGE:TAG", help="Override a single image")
+    args = parser.parse_args(argv)
+    return args
+
+
+async def get_sensor_value(client: aiokatcp.Client, sensor_name: str, sensor_type: Callable):
+    """Retrieve a sensor value from a katcp client."""
+    try:
+        _, informs = await client.request("sensor-value", sensor_name)
+    except (aiokatcp.FailReply, ConnectionError) as exc:
+        print(f"Error: {exc}", file=sys.stderr)
+        sys.exit(1)
+    return sensor_type(informs[0].arguments[4].decode())
+
+
+async def async_main(args) -> int:
+    cmc_client = await aiokatcp.Client.connect(args.cmc_host, args.cmc_port)
+
+    try:
+        reply, informs = await cmc_client.request("subordinate-list")
```

Comment:
```suggestion
        _, informs = await cmc_client.request("subordinate-list")
```
I think that'll make flake8 happier.